### PR TITLE
satellite/satellitedb: Use var block for single variable declaration

### PR DESCRIPTION
**Pull request by ifraixedes**
What: Use var block and remove some spaces at the end of lines.

Why: To have more consistent godoc format

Before this change the godoc format is rendered as

2019-07-23-135743_813x252_scrot

While in other packages we have used blocks for a single var declaration at package level. For example

2019-07-23-135804_1090x345_scrot

I consider that when there is more than block of error variables, it's easier to read that all the different variables declared in different files of the same package be defined in blocs.

Please describe the tests: N/A
Please describe the performance impact: N/A

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/satellitedb/migrate.go [hunk 1]
```diff
@@ -18,8 +18,10 @@ import (
 	"storj.io/storj/satellite/satellitedb/pbold"
 )
 
-// ErrMigrate is for tracking migration errors
-var ErrMigrate = errs.Class("migrate")
+var (
```
Comment (wthorp) [inline satellite/satellitedb/migrate.go]: I will always disagree with more lines of code

Comment (ifraixedes) [reply satellite/satellitedb/migrate.go]: It's about the godoc otherwise I would never have found it, not changed it

Comment (wthorp) [reply satellite/satellitedb/migrate.go]: ah, thanks for teaching me that.

Comment (ifraixedes) [reply satellite/satellitedb/migrate.go]: you can see the difference in the PR comment

**Pull request merged**